### ISSUE TITLE
Problems encountered when using eagle2 qwen2

### ISSUE THREAD

**Issue by lingkong-q** [sgl-project/sglang#7477]
Hello, I modified the input parameters of the sglang Engine in Verl to use the eagle2 model on the qwen2 model. However, I encountered the following problem and the program was interrupted. I don't know why.

My model is _Qwen2.5-7B-Instruct_ And the draft model is _EAGLE-Qwen2-7B-Instruct_

A great many：
`invalid eagle tree!!! Detected a token with no parent token selected. Please check if the logprob has nan. The token will be ignored to keep proceeding.`

**Comment by Swipe4057** [comment]
I would help, but there is no launch command, no environment - nothing

**Comment by lingkong-q** [comment]
The dataset used is an internal one and cannot be displayed. However, when printing the log, it was found that the logits output by the Eagle model contain NaN. I don't know why this is happening. Using the official recommended EAGLE-Qwen2-7B-Instruct model or my own trained model, the same issue occurs. I would like to ask if it is normal for the Eagle model to produce NaN?
